airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -595,6 +602,7 @@ def invoke_defer_method(self):
                 should_delete_pod=self.is_delete_operator_pod,
                 get_logs=self.get_logs,
                 startup_timeout=self.startup_timeout_seconds,
+                startup_check_interval=self.startup_check_interval_seconds,
```

Comment:
I'm not convinced this does anything. The trigger uses `poll_interval` instead. Feels a little duplicative. Maybe we deprecate `poll_interval` and just use this new one?

Reply:
Agree, it is not doing anything here. Will fix this.
Moreover, I have no strong opinion in terms of deprecating `poll_interval` or renaming the new parameter `startup_check_interval_seconds/startup_timeoutcheck_interval_seconds` to `poll_interval` in the new implementation.

Reply:
Ok, going through the code base, I came across different occurrences of unparametrized `time.sleep` calls (with either 1 or 2 seconds):

1. `airflow.providers.cncf.kubernetes.utils.pod_manager.PodManager.await_pod_start`: the startup polling, set to 1s
2. `airflow.providers.cncf.kubernetes.utils.pod_manager.PodManager.fetch_container_logs`: polling that follows the logs, set to 1s
3.  `airflow.providers.cncf.kubernetes.utils.pod_manager.PodManager.await_container_completion`: polling of container state if log following is deactivated, set to 1s
4.  `airflow.providers.cncf.kubernetes.utils.pod_manager.PodManager.await_pod_completion`: same as above for container completion, set to 2s
5. `airflow.providers.cncf.kubernetes.triggers.pod.KubernetesPodTrigger.run`: set to 2s

I think we want to change 1 but not 2 to 4. I am not sure for 5 though, because it does also check for the status once the pod is running, no? Please correct me if I am wrong.